charts/victoria-metrics-anomaly/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: victoria-metrics-anomaly
 description: Victoria Metrics Anomaly Detection - a service that continuously scans Victoria Metrics time series and detects unexpected changes within data patterns in real-time.
-version: 1.0.0
-appVersion: v1.9.2
+version: 1.0.1
```

Comment:
This change breaks compatibility with previous one in terms of configuration `model -> models` so incrementing patch version is misleading here. Could you increment feature version here instead?

Reply:
I'd also vote for it to be `1.1.0`.
@zekker6 , technically, `models` *it's a backward compatible change* (so the old-style `model` is implicitly converted to new-style `models` under the hood, but it will definitely be deprecated in future releases) - please see [header of models page for an example](https://docs.victoriametrics.com/anomaly-detection/components/models/), so what's the best action we can take for helm charts given this context?

Reply:
added to changelog